src/main/java/com/redhat/coolstore/service/ShoppingCartOrderProcessor.java

```diff
@@ -1,35 +1,33 @@
 package com.redhat.coolstore.service;
 
 import java.util.logging.Logger;
-import javax.ejb.Stateless;
-import javax.annotation.Resource;
-import javax.inject.Inject;
-import javax.jms.JMSContext;
-import javax.jms.Topic;
+import jakarta.enterprise.context.ApplicationScoped;
+import jakarta.annotation.Inject;
+import io.quarkus.micrometer.runtime.MicrometerEmitter;
```

Comment:
this I don't think should be using a emitter:
https://github.com/konveyor-ecosystem/coolstore/commit/15bde8ce3d0d76272c4cd09a13c38dcaa4dde46c#diff-5906b2697f298e225bd364be5ee26757e14e477f978ece1e46787c3f1cf4e9f1R7-R8